scripts/perftune.py

```diff
@@ -614,14 +618,18 @@ def __dev_is_hw_iface(self, iface):
     def __check_dev_is_bond_iface(self):
         if not os.path.exists('/sys/class/net/bonding_masters'):
             return False
-
-        return any([re.search(self.nic, line) for line in open('/sys/class/net/bonding_masters', 'r').readlines()])
+        bonddict={}
```

Comment:
spaces around "=". 
Something is terribly wrong with your editor style config ;).

Reply:
I will run autoformat now always before commiting code (and will recheck all diffs with it), I will use default style from Visual Studio Code (or any other you point me to or if there will be any set in seastar)